libs/openchallenges/home/src/lib/challenge-host-list/challenge-host-list.component.html

```diff
@@ -4,14 +4,12 @@
       <h3 class="section-title">Discover Organizations</h3>
       It takes a village to run a challenge - see who has been contributing their time and effort.
     </div>
-    <div class="row">
-      <div class="card-group">
-        <openchallenges-organization-card
-          *ngFor="let organizationCard of organizationCards$ | async"
-          [organizationCard]="organizationCard"
-        >
-        </openchallenges-organization-card>
-      </div>
+    <div class="card-group">
+      <openchallenges-organization-card
+        *ngFor="let organizationCard of organizationCards$ | async"
+        [organizationCard]="organizationCard"
+      >
+      </openchallenges-organization-card>
```

Comment:
Relatedly, should we add some default text for challenges with no annotated organizations?

Reply:
^ Actually, this can be discussed in a separate issue/PR.

Reply:
(Currently the comment left on the home page's html) Do you mean the scenario when the challenge has no contributors at all and default message is needed in `challenge-contributors`? 

Although, I think ^ is very rare. I agree to have some info to notify user it's not a bug :) for future work~

Reply:
> have some info to notify user it's not a bug

Yes! That was my thinking.  The first challenge I looked at had an empty page so I thought it was a bug xD

Reply:
Sounds good. I can add the default text after this pr is merged. Thanks for the suggestion.